### DISCUSSION TITLE
About the Twilio Student Pack Program

### DISCUSSION THREAD

**Discussion by steven88663** [community/community#59485] [GitHub Education]
### Hi everyone,

It seems that the Twilio student package plan and the update of the promotion code have been stopped on Github, but I still got the promotion code from the Twilio support staff. This makes me wonder why Twilio is updating this promo code, and why Github can't update and display it. Why can't Github benefit our group of students?@Twilio

**Comment by TASMEAH9** [comment]
can you plz share promo ?

**Reply by steven88663** [reply]
现在已经彻底被取消了

**Reply by TASMEAH9** [reply]
so there is no way to get promo code now ?

**Reply by steven88663** [reply]
yes

**Reply by TASMEAH9** [reply]
what do you use twilio for ? .. and can you give me your telegram ?

**Reply by HassanyasooE** [reply]
can you give us the code ?

**Reply by 1746705990** [reply]
I am going to use Github to register Twilio, once the account is registered, it is locked. I am saddened and angry that after multiple tickets it was confirmed that my account was fraudulent.🥹🥹🥹

**Reply by mannubaveja007** [reply]
> I am going to use Github to register Twilio, once the account is registered, it is locked. I am saddened and angry that after multiple tickets it was confirmed that my account was fraudulent.🥹🥹🥹

don't mind buddy it's same looks like @twilio don't want the developers to register...

**Comment by steven88663** [comment]
Can't register now?

Terracotta ***@***.***> 于2023年7月25日周二 08:04写道：

**Comment by LuukHaket** [comment]
They stopped the program cause people were abusing it, tough shit.

**Comment by steven88663** [comment]
When will it be restarted? Or are there other similar promotions?

Luuk Haket ***@***.***> 于2024年1月8日周一 18:04写道：
